backend/bitboxbase/bbbconfig/bbbconfig.go

```diff
@@ -0,0 +1,229 @@
+// Copyright 2018 Shift Devices AG
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bbbconfig
+
+import (
+	"bytes"
+	"sync"
+
+	fileconfig "github.com/digitalbitbox/bitbox-wallet-app/util/config"
+	"github.com/flynn/noise"
+)
+
+const configFilename = "bitboxbase.json"
+
+// RegisteredBase has information about BitBoxBases registered with the backend
+type RegisteredBase struct {
+	BaseID   string `json:"baseID"`
+	Hostname string `json:"hostname"`
+}
+
+// BBBConfigurationInterface provides an interface to interact with the persisted BBBConfig
+type BBBConfigurationInterface interface {
```

Comment:
I don't see what you need the interface for. Are you sure you actually need it?

Reply:
Yeah, I suppose not since it's just implemented once in the same file. I'll remove it